test/form/samples/emit-asset-file/_config.js

```diff
@@ -5,6 +5,7 @@ const assert = require('assert');
 module.exports = {
 	description: 'supports emitting assets from plugin hooks',
 	options: {
+		strictDeprecations: false,
```

Comment:
This option should only be used in tests in a `deprecated` folder. The idea is that when a deprecated feature is removed, all related tests can easily be found and deleted as well. The recommended approach here would be to duplicated the test:
- a copy of this test in its previous form using `isAsset` an d `strictDeprecations: false` is placed in `form/deprecated`
- the test in `form` is modified to use only `type` instead of `isAsset` so that when the deprecated test is deleted, the new functionality is still tested and we do not need to manually look through our tests.